site/app/models/GradingOrder.php

```diff
@@ -201,7 +201,7 @@ public function sort($type, $direction) {
      * @param bool $to_same_itempool If the next submitter should have the same itempool item as the current submitter for the targeted component
      * @return Submitter|null Previous submitter to grade
      */
-    public function getPrevSubmitter(Submitter $submitter, bool $to_ungraded = false, $component_id = "-1", $to_same_itempool = null): ?Submitter {
+    public function getPrevSubmitter(Submitter $submitter, bool $to_ungraded = false, $component_id = "-1", bool $to_same_itempool = false): ?Submitter {
```

Comment:
PHP 7.x unfortunately does not support union types, which `$component_id` is (`string|int`). However, would it be possible to replace the default value with `-1` instead of `"-1"` and get the same result?